src/pages/SignUpPage/components/SignUpForm/styles.js

```diff
@@ -1,6 +1,6 @@
 const styles = () => ({
-  signUpButton: {
-    marginTop: '10px',
+  signUp: {
+    marginTop: 8,
```

Comment:
What does 8 mean here? 8 pixels?

Reply:
yeah 8 converts to px for material ui css :)